README.md

```diff
@@ -59,8 +59,30 @@ The `solutions` folder contains complete solutions for each exercise. These will
 
 In general, you can ignore this folder, but you might find it helpful to reference if you get stuck.
 
+## Development Container
+
+This repo contains a [.devcontainer.json](.devcontainer.json) file.
+
+If you are using a Development Container compatible IDE (Eg. VS Code, IntelliJ Ultimate) and have Docker running, you can use the `.devcontainer.json` file to automatically configure your development environment with the required settings.
```

Comment:
```suggestion
If you are using a Development Container compatible IDE (Eg. VS Code, IntelliJ Ultimate) and have Docker running, you can use the `.devcontainer.json` file to automatically configure your development environment with the required settings. This includes automatically setting up the correct version of the JVM, installing Maven, and potentially configuring your IDE with any required plugins. It will give you a fully functional development environment in moments.
```